teuthology/openstack/__init__.py

```diff
@@ -183,6 +183,7 @@ class OpenStack(object):
     image2url = {
         'centos-6.5': 'http://cloud.centos.org/centos/6/images/CentOS-6-x86_64-GenericCloud-1508.qcow2',
         'centos-7.0': 'http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1508.qcow2',
+        'centos-7.1': 'http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1508.qcow2',
```

Comment:
Ok. Feels weird to point '7.0' and '7.1' to the same image... but it's not like centos.org is giving us a ton of choice here.